src/uniprotkb/components/entry/ComputationallyMappedSequences.tsx

```diff
@@ -15,15 +15,20 @@ import apiUrls from '../../../shared/config/apiUrls';
 import { Location, LocationToPath } from '../../../app/config/urls';
 
 import { MessageLevel } from '../../../messages/types/messagesTypes';
+import { Sequence } from '../../../shared/types/sequence';
+import { OrganismData } from '../../adapters/namesAndTaxonomyConverter';
 
-// NOTE: Jie told me this would be replaced by a different
-// format he called "FASTA" entry
 type ProteinEntryLight = {
-  accession: string;
+  id: string;
+  sequence: Sequence;
   entryType: string;
-  sequenceLength: number;
+  uniProtkbId: string;
+  proteinName: string;
+  organism: OrganismData;
   geneName: string;
-  geneNameType: string;
+  proteinExistence: string;
+  flagType: string;
```

Comment:
Same thing here (also, what is this? Could you add a comment above to explain)

Reply:
Just checked with Jie, `flagType` will be removed as the value should only ever be "Fragment". Currently we also have "Precursor" but that is a bug and should be removed at the next release. Removing it from the type def.